README.md

```diff
@@ -8,6 +8,10 @@ and SYS files) typically used on Windows.  It's intended to help with questions
 * Why is this binary so big?
 * What can be done to make it smaller?
 
+<p align="center">
+    <img alt="Screenshot of SizeBench, showing wasted virtuals analysis" src="https://github.com/microsoft/SizeBench/blob/main/EndUserDocs/Images/WastefulVirtuals_Overview.png?raw=true" width="600" />

```

Comment:
Thank you for your contribution!  Could you format this as markdown instead of HTML?

GitHub markdown for images looks like this:

![Alt Text](/path/to/image.png)

I think?

Reply:
I used the HTML version so I could limit the size. Otherwise the image was too big.

https://stackoverflow.com/questions/37534560/how-to-i-change-the-size-of-an-image-in-github-markdown